### PR TITLE
configured multitiered neural-nets

### PR DESCRIPTION
**Pull request by earthbound19**
Added many tech verbs from various sources (and originally).

It would be nice to use something like https://www.npmjs.com/package/conjugate to suit verbs to tenses and pronouns in the random creation of full sentences. Maybe I'll make a node module that does that.

### CODE REVIEW

**Comment by atomantic** [comment]
awesome, I'll check this out tonight. Thanks for the PR! Adding a conjugation tie in would be great. Might be worth adding a sub-module so the buzzphrase could be requested in conjugated form.

File: data/verbs.js [hunk 1]
```diff
@@ -1,47 +1,240 @@
...
+  ,"benchmarked"
+  ,"blocked"
+  ,"booted"
+  ,"branded"
+  ,"changed"
+  ,"clicked"
+  ,"clicked-on"
+  ,"closed"
+  ,"clustered"
+  ,"codeed"
```
Comment (atomantic) [inline data/verbs.js]: is this intended to be `coded`?

Comment (earthbound19) [inline data/verbs.js]: Yes. I can fix that in my repo and you can do some magic to pull my latest and not this; or you can fix that after merging, if you wish.

Comment (earthbound19) [inline data/verbs.js]: Wait. Unless we are talking about employees and codees :)

**Comment by earthbound19** [comment]
Woohoo for the merge! I just noticed from output of this :p that I introduced a typo of "sortd" which is supposed to be "sorted" (in addition to the "codeed") which you mentioned. It may be worth pasting the code into a spell-checker.

**Comment by atomantic** [comment]
Hah. That idea did cross my mind. I'll fix this in the next hour (on the bus). Would you like to have contributor access? Given the other random/Markov generators on your site, you clearly have an interest in it :)

**Comment by earthbound19** [comment]
I'd be happy to have contributor access, thanks! I won't guarantee any contributions at the moment, but I'll keep the option open.

**Pull request merged**